cpp/include/eidecl_graph.hpp

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+namespace cugraph {
+extern template class GraphViewBase<int32_t, int32_t, float>;
+extern template class GraphViewBase<int32_t, int32_t, double>;
+extern template class GraphViewBase<int32_t, int64_t, float>;
+extern template class GraphViewBase<int32_t, int64_t, double>;
+extern template class GraphViewBase<int64_t, int32_t, float>;
```

Comment:
The combination <int64_t, int32_t, *> is a combination we should avoid through the code in this PR.

I think it increases the number of combinations we need to build into the library with no real benefit.  The only way <int64_t, int32_t, *> would be useful is if the average degree of vertices in the graph < 2 and the number of edges is close to 2^31-1.  I've never encountered a graph with these characteristics that would actually be meaningful to analyze.

I think in the unlikely event of this case, requiring the user to go to <int64_t, int64_t, *> is not unreasonable.

Note this pattern recurs below in this file and in several other files.

Reply:
But that's exactly what the code says: " _supress_ those instantiations by using `extern` on them". If those would have been in `eidir_*.hpp` then that would've been a problem.

Reply:
But, done (for the purpose of symmetry of eidir/eidecl...which really should not necessarily be symmetric; their structure is driven by different needs).